src/views/admin/PendingAccounts.vue

```diff
@@ -9,9 +9,10 @@
           <loading v-model:active="isLoading"
                :can-cancel="false"
                :is-full-page= "false"/>
+          <p class="notification is-danger is-light" v-if="hasLoadingError">Liste des comptes en attente de validation indisponible</p>
```

Comment:
For the formulation, we need to be more explicit and tell that an unexpected error happened. Maybe even apologies ? And give an action. Try to reload or contact your administrator.

Reply:
Ok, I took your suggestion from the RC to the word I thought that's what you thought about in the first place.
I will make a more explicit explanation of the error.

Reply:
Changed.